client/src/utils/filePermissions.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * Check permissions for a selected file or files
+ * @param {String} name
+ * @param {Array<Object>} files
+ * @param {String} scheme
+ * @param {String} api
+ * @returns {Boolean}
+ */
+export default function getFilePermissions(name, { files, scheme, api }) {
+  const isPrivate = ['projects', 'private'].includes(scheme);
+  const isArchive = files[0]
+    ? files[0].name.includes('.zip') || files[0].name.includes('tar.gz')
```

Comment:
```suggestion
    ? files[0].name.includes('.zip') || files[0].name.includes('.tar.gz')
```

Reply:
I recommend endsWith instead of includes.